packages/annotation-converter/test/converterTest.spec.ts

```diff
@@ -865,27 +865,27 @@ describe('Annotation Converter', () => {
         const convertedTypes = convert(parsedMetadata);
 
         function getAction(name: string) {
-            const result = convertedTypes.actions.find((action) => action.fullyQualifiedName === name);
+            const result = convertedTypes.actions.by_fullyQualifiedName(name);
             expect(result).toBeDefined();
             return result;
         }
 
         const dataFields1 = convertedTypes.entityTypes[0]?.annotations.UI?.LineItem as any[];
         expect(dataFields1[0].ActionTarget).toBe(getAction('TestService.action(TestService.Entity1)'));
         expect(dataFields1[1].ActionTarget).toBe(getAction('TestService.function(TestService.Entity1)'));
-        expect(dataFields1[2].ActionTarget).toBe(getAction('TestService.action'));
-        expect(dataFields1[3].ActionTarget).toBe(getAction('TestService.function'));
+        expect(dataFields1[2].ActionTarget).toBe(getAction('TestService.action()'));
```

Comment:
Shouldn't the non () syntax still work ?

Reply:
this should be covered by https://github.com/SAP/open-ux-odata/blob/01bcc25246419df9cf3bbe160ef04efe9087af1a/packages/annotation-converter/test/converterTest.spec.ts#L992-L1032

Reply:
Ok then